tests/verify_override_true_profile.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import argparse
+import lxml.etree as ET
+from sys import stderr
+
+
+TITLES_PATH = "//*[local-name()=\"Profile\"]/*[local-name()=\"title\"][not(@override=\"true\")]"
```

Comment:
Please rework the XPath expressions to use namespaces instead of using the `local-name` construction. Using the `local-name` function was a workaround for the situation that the `xmllint` utility doesn't support namespaced XMLs, but Python`xmltree` supports namespaced XMLs, so we don't need this workaround.